### ISSUE TITLE
When do you think you should be ready for production?

### ISSUE THREAD

**Issue by marcus7777** [GoogleWebComponents/firebase-element#64]
I've been struggling to get the firebase element working for about 6 months now and I'm still struggling sometimes just writing data to new location or knowing whether to use collection or document it's still confusing. I got Scott miles to help me out at the polymer conference and it seem to be working then and now I'm back home and I still have the same problem. I'm thinking at the moment I should write my JavaScriptAPI from firebase directly and not try to use this element is that right?

**Comment by devAtPolydeals** [comment]
:+1:  Amen to this my friend....

**Comment by marcus7777** [comment]
demo http://jsbin.com/tufeva/edit?html,console,output

**Comment by ebidel** [comment]
Agreed, the docs for these elements could definitely be better but there are a couple of things wrong with your  bin:

- you have a dom-module and import defined outside of your `<html>`
- you should bind the input to `{{data.hi::input}}`
- `firebase-document` has a `location` property. Not a `url` property.
- your demo will redefine `data.hi` on every page load. I think you probably want to use the value if one exists in FB...?

http://jsbin.com/rocelexuya/edit?html,output is more of what you want, although still not perfect. I'm no FB expert.

**Comment by cdata** [comment]
I will take a look.
